integration-tests/src/test/java/com/datastax/oss/driver/api/core/cql/BoundStatementIT.java

```diff
@@ -151,6 +164,68 @@ public void should_allow_custom_codecs_when_setting_values_in_bulk() {
     }
   }
 
+  @Test
+  public void should_propagate_attributes_when_preparing_a_simple_statement() {
```

Comment:
Should add `@CassandraRequirement` for 2.2 since Custom Payload won't work with C* 2.1.  Either that or don't include custom payload if not supported by protocol version (similar to `supportsPerRequestKeyspace`)